### 1. Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].Найдите ее среднее значение и дисперсию.

In [1]:
a=200
b=800

print(f'среднее значение равно {(a+b)/2}, а дисперсия {(b-a)**2/12}')

Ответ: среднее значение  равно 500, а дисперсия 30000

_______________________________________________________________________________________________________________________________

### 2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.

In [2]:
D=0.2
a=0.5
b= 2.4**0.5-a
print(f'Правая граница величины B примерно {b}, тогда среднее значение равно {(a+b)/2}')

Ответ: Правая граница величины B примерно 1.0491933384829668, тогда среднее значение равно 0.7745966692414834

_______________________________________________________________________________________________________________________________

### 3. Непрерывная случайная величина X распределена нормально и задана плотностью                распределения f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-(x+2)**2) / 32). 
### Найдите:
### а). M(X)
### б). D(X) 
### в). std(X) (среднее квадратичное отклонение)

Ответ: M(X)=-2, D(X)=16, std(X)=4 

_______________________________________________________________________________________________________________________________

### 4. Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см
б). больше 190 см
в). от 166 см до 190 см
г). от 166 см до 182 см
д). от 158 см до 190 см
е). не выше 150 см или не ниже 190 см
ё). не выше 150 см или не ниже 198 см
ж). ниже 166 см.

In [3]:
# Воспользуемся таблицей нормального распределения
mu=174
print('Вероятность того, что выбранный взрослый человек имеет рост больше 182 см. - 0.1587 ')
print('Вероятность того, что выбранный взрослый человек имеет рост больше 190 см. - 0.0228 ')
print(f'Вероятность, что выбранный взрослый человек имеет рост от 166 см до 190 см - {0.9772-0.1587}')
print(f'Вероятность, что выбранный взрослый человек имеет рост от 166 см до 182 см - {0.8413-0.1587}')
print(f'Вероятность, что выбранный взрослый человек имеет рост от 158 см до 190 см - {0.9772-0.0228}')
print(f'Вероятность, что выбранный взрослый человек имеет рост не выше 150 см или не ниже 190 см - {0.013+0.0228 }')
print(f'Вероятность, что выбранный взрослый человек имеет рост не выше 150 см или не ниже 198 см - {0.013*2 }')
print(f'Вероятность, что выбранный взрослый человек имеет рост ниже 166 см. - 0.1587')

Вероятность того, что выбранный взрослый человек имеет рост больше 182 см. - 0.1587 
Вероятность того, что выбранный взрослый человек имеет рост больше 190 см. - 0.0228 
Вероятность, что выбранный взрослый человек имеет рост от 166 см до 190 см - 0.8185
Вероятность, что выбранный взрослый человек имеет рост от 166 см до 182 см - 0.6826000000000001
Вероятность, что выбранный взрослый человек имеет рост от 158 см до 190 см - 0.9543999999999999
Вероятность, что выбранный взрослый человек имеет рост не выше 150 см или не ниже 190 см - 0.0358
Вероятность, что выбранный взрослый человек имеет рост не выше 150 см или не ниже 198 см - 0.026
Вероятность, что выбранный взрослый человек имеет рост ниже 166 см. - 0.1587


Ответ: 
 Вероятность того, что выбранный взрослый человек имеет рост больше 182 см. - 0.1587 
 
Вероятность того, что выбранный взрослый человек имеет рост больше 190 см. - 0.0228 

Вероятность, что выбранный взрослый человек имеет рост от 166 см до 190 см - 0.8185

Вероятность, что выбранный взрослый человек имеет рост от 166 см до 182 см - 0.6826

Вероятность, что выбранный взрослый человек имеет рост от 158 см до 190 см - 0.9544

Вероятность, что выбранный взрослый человек имеет рост не выше 150 см или не ниже 190 см - 0.0358

Вероятность, что выбранный взрослый человек имеет рост не выше 150 см или не ниже 198 см - 0.026

Вероятность, что выбранный взрослый человек имеет рост ниже 166 см. - 0.1587

### 5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [4]:
std=25**0.5
print(f'Рост человека, равный 190 см, отклоняется от математического ожидания на {(190-178)/std} сигмы')

Рост человека, равный 190 см, отклоняется от математического ожидания на 2.4 сигмы


Ответ: Рост человека, равный 190 см, отклоняется от математического ожидания на 2.4 сигмы